plugin/geoip/README.md

```diff
@@ -40,6 +40,18 @@ geoip [DBFILE]
 
 * **DBFILE** the mmdb database file path.
 
+By default, geoip plugin doesn't honor EDNS0 subnet extension, but it can be enabled using the extended syntax:
+
+```
+geoip [DBFILE] {
+    edns-subnet-processing
+}
+```
+
+* **DBFILE**: see above
```

Comment:
**Nit**

I find adding once again here DBFILE argument a bit redundant, given that is already defined above. I also changed the wording along with syntax and examples section... What do you think?

## Syntax
```txt
geoip [DBFILE] {
    edns-subnet-processing
}
```

* **DBFILE** the mmdb database file path.
* `edns-subnet-processing` By default, geoip plugin read client IP and ignores EDNS0 subnet extension, using this option enable reading source IP from [EDNS0 subnet](https://en.wikipedia.org/wiki/EDNS_Client_Subnet) when available. This help identifying closest source IP address as well as making GeoIP test processing easy: `dig +subnet=1.2.3.4 @dns-server.example.com www.geo-aware.com`.

## Examples

The following configuration configures the `City` database and enables EDNS subnet processing.

```txt
. {
    geoip /opt/geoip2/db/GeoLite2-City.mmdb {
        edns-subnet-processing
    }
    metadata # Note that metadata plugin must be enabled as well.
}
```

Reply:
I think it's a good idea, but I'd mention that `edns-subnet-processing` is optional, as well as the { } block. Otherwise, I'm happy with the rewording.

Reply:
From my side, sounds good to be explicit with the optional nature of the argument.

Reply:
Please have a look, and open a conversation if something else is needed.